packages/interface-ipfs-core/src/add.js

```diff
@@ -483,7 +486,7 @@ module.exports = (common, options) => {
       expect(expectedResult.err).to.not.exist()
       expect(result[0].cid.toString()).to.equal(expectedResult[0].cid.toString())
       expect(result[0].size).to.equal(expectedResult[0].size)
-      expect(result[0].path).to.equal(text)
+      // expect(result[0].path).to.equal(text)
```

Comment:
Should this be uncommented?

Reply:
same reason as below, path is `download` now and not the text, i will remove the comment